book/src/super-sql/operators/join.md

```diff
@@ -64,8 +64,8 @@ within nested scopes, the pipeline join operator uses dataflow scoping to join d
 Here, all data is combined into joined records that can be operated upon 
 like any other record without complex scoping logic.
 
-If relational scoping is desired, the [SQL join](../sql/join.md)
-operator can be used instead.
+If relational scoping is desired, a [SQL `JOIN` clause](../sql/join.md)
```

Comment:
It seems like capitalizing the SQL keywords would help emphasize when we're talking about SQL.